### PR TITLE
PE: Fixed section alignment calculation [Fix #6353]

### LINKED ISSUE #6353: [PE32] Wrong address -> section calculation
[This](http://spin.quequero.org/uicwiki/images/CM01.zip) crackme causes some section calculation issues: the PE plugin loads the section table correctly:

```
[0x00401000]> S
[00] * 0x00000600 mr-x va=0x00401000 sz=0x0600 vsz=0x2000 CODE
[01] . 0x00000c00 mrw- va=0x00402000 sz=0x0200 vsz=0x2000 DATA
[02] . 0x00000e00 mrw- va=0x00403000 sz=0x0800 vsz=0x2000 .idata
[03] . 0x00001600 mr-- va=0x00404000 sz=0x0200 vsz=0x2000 .edata
[04] . 0x00001800 ---- va=0x00405000 sz=0x0200 vsz=0x2000 .reloc
[05] . 0x00001a00 ---- va=0x00406000 sz=0x1400 vsz=0x3000 .rsrc

[0x00401000]> S.
0x00401600 0x00401c00 CODE
```

but if I seek to other sections, radare prints the wrong section:

```
[0x00402000]> S.
0x00405600 0x00405800 .edata

[0x00403000]> S.
0x00407a00 0x00408e00 .rsrc

[0x00404000]> S.
0x00407a00 0x00408e00 .rsrc

[0x00405000]> S.
0x00407a00 0x00408e00 .rsrc

[0x00406000]> S.
0x00407a00 0x00408e00 .rsrc

```

This issue affects even the disassembler listing because it can't find data and strings in "DATA" segment:

```
[0x00401000] pdf
...
│           0x00401097      ff35ca204000   push dword [0x4020ca]
│           0x0040109d      6a00           push 0
│           0x0040109f      6a00           push 0
│           0x004010a1      6800800000     push 0x8000
│           0x004010a6      6800800000     push 0x8000
│           0x004010ab      6a6e           push 0x6e      
│           0x004010ad      68b4000000     push 0xb4
│           0x004010b2      680000cf00     push 0xcf0000
│           0x004010b7      68e7204000     push 0x4020e7    ; Non detected window title
│           0x004010bc      68f4204000     push 0x4020f4    ; Non detected window's class name
│           0x004010c1      6a00           push 0
│           0x004010c3      e8cc030000     call sub.USER32.dll_CreateWindowExA_494
...
```

### PR DESCRIPTION
**Pull request by Dax89**
Fixes https://github.com/radare/radare2/issues/6353

### CODE REVIEW

**Comment by Maijin** [comment]
Use "Fix #XXXX" in body or title

File: libr/bin/format/pe/pe.c [hunk 1]
```diff
@@ -111,6 +110,15 @@ ut64 PE_(r_bin_pe_get_image_base)(struct PE_(r_bin_pe_obj_t)* bin) {
 	return imageBase;
 }
 
+static PE_DWord bin_pe_calc_alignment(RBinPEObj* bin, PE_DWord value) {
```
Comment (radare) [inline libr/bin/format/pe/pe.c]: that indentation hurts. sorry to merge without noticing this. gotta fix in master now

Comment (Dax89) [inline libr/bin/format/pe/pe.c]: Ooops, sorry, vim fooled me.
I will send another PR now!

**Comment by radare** [comment]
nono, its already fixed in master

**Comment by Maijin** [comment]
@Dax89 Can you add a test based on your crackme in radare2-regressions please.

**Comment by Dax89** [comment]
@Maijin ok!

**Pull request merged**